app/api/leaderboard/functions.ts

```diff
@@ -113,3 +113,26 @@ export default async function fetchGitHubReleases(
     )
     .slice(0, sliceLimit);
 }
+
+export async function fetchAllReposName() {
+  const response = await octokit.graphql({
+    query: `
+        query GetTop10ActiveRepos($org: String!) {
+          organization(login: $org) {
+            repositories(first: 20, orderBy: { field: UPDATED_AT, direction: DESC }) {
+              nodes {
+                name
+              }
+            }
+          }
+        }
+      `,
+    org: env.NEXT_PUBLIC_GITHUB_ORG,
+  });
+
+  const repos = (
+    response as ReleasesResponse
+  ).organization.repositories.nodes.map((repo) => repo.name);
+
+  return ["All", ...repos.flat()];
```

Comment:
Since this is a generic function, pushing "All" to the top may not be desired when called from other places in the app (if re-used later on by something else...)

Inject all from the place where this function is called

```suggestion
  return repos.flat();
```